src/script/lua_api/l_inventory.cpp

```diff
@@ -273,18 +270,17 @@ int InvRef::l_set_lists(lua_State *L)
 	}
 
 	// Make a temporary inventory in case reading fails
```

Comment:
Is this even necessary?  I don't see how inventory list reading could fail in such a manner that necessitates canceling the entire operation.

Plus, read_inventory_list has no return value, meaning there's no condition to abort the operation on anyway.